aws_lambda_builders/workflows/nodejs_npm/utils.py

```diff
@@ -48,3 +66,189 @@ def abspath(self, path):
 
     def is_windows(self):
         return platform.system().lower() == "windows"
+
+    def walk(self, dir, topdown=True):
+        return os.walk(dir, topdown)
+
+
+class DependencyUtils(object):
+    """
+    Collection of helper functions for managing local NPM dependencies
+    """
+
+    @staticmethod
+    def ship_module(package_path, from_dir, to_dir, osutils, subprocess_npm):
+        """
+        Helper function to package a module and extract it to a new directory
+
+        Parameters
+        ----------
+        package_path : str
+            Package path
+        from_dir : str
+            Origin directory path
+        to_dir : str
+            Destination directory path
+        osutils : OSUtils
+            OS utils
+        subprocess_npm : SubprocessNpm
+            NPM process
+        """
+        tarfile_name = subprocess_npm.run(["pack", "-q", package_path], cwd=from_dir).splitlines()[-1]
```

Comment:
Would this make sense in its own method? It's being used elsewhere, and a comment as to why `-1` is correct would help with clarity.